packages/sanity/src/core/releases/tool/detail/review/DocumentReviewHeader.tsx

```diff
@@ -1,67 +1,76 @@
 import {type PreviewValue, type SanityDocument} from '@sanity/types'
-import {AvatarStack, Box, Flex} from '@sanity/ui'
+import {AvatarStack, Box, Card, Flex} from '@sanity/ui'
 
 import {RelativeTime} from '../../../../components/RelativeTime'
 import {UserAvatar} from '../../../../components/userAvatar/UserAvatar'
 import {type BundleDocument} from '../../../../store/bundles/types'
 import {Chip} from '../../../components/Chip'
 import {ReleaseDocumentPreview} from '../../../components/ReleaseDocumentPreview'
+import {type DocumentValidationStatus} from '../bundleDocumentsValidation'
 
 export function DocumentReviewHeader({
   previewValues,
   document,
   isLoading,
   history,
   release,
+  validation,
 }: {
   document: SanityDocument
   previewValues: PreviewValue
   isLoading: boolean
   release: BundleDocument
+  validation?: DocumentValidationStatus
   history?: {
     createdBy: string
     lastEditedBy: string
     editors: string[]
   }
 }) {
   return (
-    <Flex justify="space-between" align="center">
-      <Box padding={1} style={{flex: '1'}}>
-        <ReleaseDocumentPreview
-          documentId={document._id}
-          documentTypeName={document._type}
-          releaseSlug={release.slug}
-          previewValues={previewValues}
-          isLoading={isLoading}
-        />
-      </Box>
-      <Flex gap={2} padding={3} style={{flexShrink: 0}}>
-        {history?.createdBy && (
-          <Chip
-            avatar={<UserAvatar size={0} user={history?.createdBy} />}
-            text={
-              <span>
-                Created <RelativeTime time={document._createdAt} useTemporalPhrase />
-              </span>
-            }
+    <Card
+      tone={validation?.hasError ? 'critical' : 'default'}
+      radius={3}
+      style={{marginBottom: 20}}
+    >
```

Comment:
This was added to this component to wrap it and show the validation error, only change here.